tests/integration/devtools/test_devtools_ui.py

```diff
@@ -256,4 +256,4 @@ def create_an_alternative_response():
     )
 
     driver.get(dash_thread_server.url)
-    driver.find_element_by_id("alternative_id")
+    dash_br.find_element("alternative_id", attribute="ID")
```

Comment:
```suggestion
    dash_br.find_element("#alternative_id")
```
I don't mind giving folks the option to use `ID` and others, but within this repo, when a CSS selector is easy I'd rather we standardize.

Reply:
I totally agree. Using the ID css selector totally slipped my mind. Great pickup @alexcjohnson. I've made that change.